<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Run AICP (command line configuration) -->
  <node pkg="aicp" type="aicp_ros_online" name="aicp_ros_online" output="screen">

    <!-- Debug mode or robot -->
    <param name="working_mode"               value="debug" />
    
    <!-- Fixed frame (default: map, debug: map_test -->
    <param name="fixed_frame"                value="map_test" />

    <!-- Prior map info -->
    <param name="load_map_from_file"         value="false" />
    <param name="localize_against_map"       value="false" />

    <!-- Reference update policy -->
    <param name="failure_prediction_mode"    value="false" />
    <param name="reference_update_frequency" value="5" />
    
    <!-- 3D point cloud characteristics -->
    <param name="batch_size"                 value="7" />
  	
  	<!-- Visualize and store -->
    <param name="verbose"                    value="true" />

  </node>

</launch>
